fastly/resource_fastly_tls_subscription_test.go

```diff
@@ -22,14 +22,15 @@ func init() {
 func TestAccResourceFastlyTLSSubscription(t *testing.T) {
 	name := acctest.RandomWithPrefix(testResourcePrefix)
 	domain := fmt.Sprintf("%s.test", name)
```

Comment:
Could we rename this for the sake of my OCD 🙂 (and also update the `testAccResourceFastlyTLSSubscriptionConfig` to use `domain1` as well please)
```suggestion
	domain1 := fmt.Sprintf("%s.test", name)
```